Aman-Kumar/Aman-Kumar.md

```diff
@@ -0,0 +1,377 @@
+---
+id: "45fc7360-fdae-ec11-983f-0022481856af"
+
+category: "Developers"
+role:"Sample Profile"
```

Comment:
This should be updated ;)

Reply:
I've committed my changes, please review them